app/src/main/java/com/pvp/app/ui/common/Modifiers.kt

```diff
@@ -0,0 +1,24 @@
+package com.pvp.app.ui.common
+
+import androidx.compose.foundation.background
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.core.graphics.toColorInt
+
+
+fun Modifier.backgroundGradient(
```

Comment:
Not sure if this should stay in `Modifiers.kt` file... Maybe? If we are going to create more personal modifiers.